### PR TITLE
NDS-140 Full bleed component

### PR DESCRIPTION
**Pull request by wa-rren-dev**
https://nicedigital.atlassian.net/browse/NDS-140

### CODE REVIEW

File: components/nds-full-bleed/README.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+```jsx
+import { FullBleed } from "@nice-digital/nds-full-bleed";
+import { Grid, GridItem } from "@nice-digital/nds-grid";
+<FullBleed>
+	<div className="container">
+		<Grid>
+			<GridItem cols={12}>
+				<p>This content is aligned to the grid</p>
+			</GridItem>
+		</Grid>
```
Comment (ediblecode) [inline components/nds-full-bleed/README.md:45]: This is missing a closing div

File: components/nds-full-bleed/README.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+#### children
+
+- Type: `React.ReactNode`
+
+The children will make up the contents of the full bleed component.
+
+#### className
+
+- Type: `React.ReactNode`
```
Comment (ediblecode) [inline components/nds-full-bleed/README.md:59]: This type should be string

File: components/nds-full-bleed/scss/full-bleed.scss [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  'medium': 3,
+  'large': 6
+);
+
+.full-bleed {
+  background-color: $nds-colour-nice-white;
+  background-position: center center;
+  background-size: cover;
+  left: 50%;
+  margin-left: -50vw;
```
Comment (ediblecode) [inline components/nds-full-bleed/scss/full-bleed.scss:15]: Does this work without the margin right? Just curious as hero has one:

https://github.com/nice-digital/nice-design-system/blob/36a76d866bee896e6779ddc3a7cc89ec8e4656bf/components/nds-hero/scss/hero.scss#L12-L15

File: components/nds-full-bleed/src/FullBleed.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+			)}
+			{...rest}
+		>
+			{children}
+		</div>
+	);
+};
+
+FullBleed.propTypes = {
+	children: PropTypes.node,
```
Comment (ediblecode) [inline components/nds-full-bleed/src/FullBleed.js:34]: Does this need to be isRequired?

File: components/nds-full-bleed/README.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+	<div className="container">
+		<Grid>
+			<GridItem cols={12}>
+				<p>This content is aligned to the grid</p>
+			</GridItem>
+		</Grid>
+</FullBleed>
+```
+
+### Props
```
Comment (ediblecode) [inline components/nds-full-bleed/README.md:49]: Is it worth mentioning that any other props are spread over the containing div like we do for other components?

**Review by ediblecode — CHANGES_REQUESTED** [review]

File: components/nds-full-bleed/scss/full-bleed.scss (hunk 3 above)
Comment (wa-rren-dev) [reply components/nds-full-bleed/scss/full-bleed.scss:15]: it does!

Screenshot 2021-03-09 at 09 51 51

**Pull request merged**